### PR TITLE
[RHDEVDOCS-5040] Bump up image folder versions for pipelines 1.10 release

### PR DESCRIPTION
**Pull request by SNiemann15**
**Don't merge before Pipelines 1.10 GA planned for April 13!**

Version(s): 4.11+

Issue: https://issues.redhat.com/browse/RHDEVDOCS-5040

Link to docs preview:
https://58250--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html
https://58250--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/using-pipelines-as-code.html#installing-pipelines-as-code-cli_using-pipelines-as-code

QE review: @ppitonak / @rupalibehera 
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/op-installing-pipelines-as-code-cli.adoc [hunk 1]
```diff
@@ -9,13 +9,13 @@
 [role="_abstract"]
 Cluster administrators can use the `tkn pac` and `opc` CLI tools on local machines or as containers for testing. The `tkn pac` and `opc` CLI tools are installed automatically when you install the `tkn` CLI for {pipelines-title}.
 
-You can install the `tkn pac` and `opc` version `1.9.1` binaries for the supported platforms:
+You can install the `tkn pac` and `opc` version `1.10.1` binaries for the supported platforms:
```
Comment (ppitonak) [inline modules/op-installing-pipelines-as-code-cli.adoc:12]: This should be 1.10.0

**Review by ppitonak — CHANGES_REQUESTED** [review]

**Comment by rupalibehera** [comment]
LGTM

**Review by ppitonak — APPROVED** [review]
lgtm

**Review by shipsing — APPROVED** [review]
LGTM @SNiemann15

**Comment by shipsing** [comment]
A generic question, if i look at the compatibility matrix here: http://dashboard.apps.cicd.ospqa.com/releases/componentmatrix/, i see the compatible OCP versions for pipelines 1.10 are 4.11, 4.12, 4.13. In the PR, do we need to change the version for cherry-picking to 4.11+, instead of 4.12+ ?

**Review by concaf — APPROVED** [review]

**Pull request merged**